helm-system-packages-portage.el

```diff
@@ -256,26 +232,13 @@ The caller can pass the list of EXPLICIT packages to avoid re-computing it."
 (defvar helm-system-packages-portage-use-source
   (helm-build-in-buffer-source "USE flags"
     :init 'helm-system-packages-portage-use-init
-    :candidate-transformer 'helm-system-packages-portage-use-transformer
```

Comment:
I see you moved the transformer to the `-use-init` function.  Why?
As I understand it, the USE flags won't be updated on change.
Let me know if I'm wrong.

Reply:
I'm unsure of how to approach this one - ideally, we want the USE-flag properties to be re-cached in `helm-system-packages-portage-refresh`, since then they'll be updated after running commands etc. However, that will inextricably link the package list source and the USE-flag source, preventing then from potentially being used independently by a third party. Thoughts?

Reply:
OK, so if I understand correctly, by moving the transformer to the init function you are updating the USE source everytime?
If this is fast enough, then I think this is the right approach.